content/en/docs/components/pipelines/v2/installation/quickstart.md

```diff
@@ -19,40 +18,41 @@ This tutorial helps you get started with a KFP deployment and a pipeline created
 
 Before you begin, you need the following prerequisites:
 
-  * **An existing Kubernetes cluster**: If you don't have a Kubernetes cluster, see [Installation][installation] for instructions about how to get one.
-  
-  * **The [kubectl](https://kubernetes.io/docs/tasks/tools/) command-line tool**: Install and configure your [kubectl context](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to connect with your cluster.
-  
-  * Run the following script to install the KFP SDK:
-    ```shell
-    pip install kfp --pre
-    ```
+* **An existing Kubernetes cluster**: If you don't have a Kubernetes cluster, see [Installation][installation] for instructions about how to get one.
 
-    **Note:** This command installs KFP v2, which is in pre-release stage and is not yet stable. The v2 documentation is being developed continually and some of the links to the v2 documentation might be unavailable.
+* **The [kubectl](https://kubernetes.io/docs/tasks/tools/) command-line tool**: Install and configure your [kubectl context](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to connect with your cluster.
+
+* Run the following script to install the KFP SDK:
+  ```shell
+  pip install kfp --pre
+  ```
+
+  **Note:** This command installs KFP v2, which is in pre-release stage and is not yet stable. The v2 documentation is being developed continually and some of the links to the v2 documentation might be unavailable.
 
 After you complete the prerequisites, click each step to view the instructions:
 
 <details>
-  <summary><a name="kfp_qs_step1"></a><h2 style="display:inline;">Step 1: Deploy a KFP standalone instance into your cluster</h2></summary>
-  <hr/>
-  This step shows how to deploy a KFP standalone instance into an existing Kubernetes cluster.
+<summary><a name="kfp_qs_step1"></a><h2 style="display:inline;">Step 1: Deploy a KFP standalone instance into your cluster</h2></summary>
 
-  Run the following script after replacing `PIPELINE_VERSION` with the desired version of KFP:
+This step demonstrates how to deploy a KFP standalone instance into an existing Kubernetes cluster.
 
-  ```shell
-  export PIPELINE_VERSION="2.0.0-alpha.4"
+Run the following script after replacing `PIPELINE_VERSION` with the desired version of KFP (release are listed [here][pypi]):
```

Comment:
The KFP BE version number is actually decoupled from the KFP SDK version number, so not sure we should include this link.

Reply:
Doh. Duh. Nice catch! Updated it to point to https://github.com/kubeflow/pipelines/releases?q=Version&expanded=true.